packages/hash/api/src/graphql/resolvers/linkedAggregation/createLinkedAggregation.ts

```diff
@@ -5,18 +5,17 @@ import {
 } from "../../apiTypes.gen";
 import { Entity, UnresolvedGQLLinkedAggregation } from "../../../model";
 import { LoggedInGraphQLContext } from "../../context";
+import { DBAdapter } from "../../../db";
+import __User from "../../../model/user.model";
 
-export const createLinkedAggregation: Resolver<
-  Promise<UnresolvedGQLLinkedAggregation>,
-  {},
-  LoggedInGraphQLContext,
-  MutationCreateLinkedAggregationArgs
-> = async (
-  _,
-  { sourceAccountId, sourceEntityId, path, operation },
-  { dataSources, user },
-) =>
-  dataSources.db.transaction(async (client) => {
+export const createLinkedAggregationRaw: (
```

Comment:
Can we call the existing resolver from mock data instead of creating this `Raw` function? They look almost identical.

Reply:
Hey Ciaran, the context parameter for the GraphQL resolver is much more complex, requiring `dataSources, emailTransporter, uploadProvider, passport, logger` along with the user object.

I'm simplifying the parameters significantly by creating a raw version of the resolver only accepting the required types of `user` and `db`.